test/vdom.test.js

```diff
@@ -659,3 +659,49 @@ testTrees("update element with dynamic props", [
     html: `<input type="text" value="foo">`
   }
 ])
+
+testTrees("elements with falsy values", [
```

Comment:
@maxholman Is this your original test? 😄 

Make sure to run prettier on all the files 🙏

Reply:
I wrote the test just for this PR yes. Has now been "prettified" :)